alicloud/resource_alicloud_polardb_account_test.go

```diff
@@ -0,0 +1,132 @@
+	name := fmt.Sprintf("tf-testAccdbaccount-%d", rand)
+	var basicMap = map[string]string{
+		"db_cluster_id":    CHECKSET,
+		"account_name":     "tftestnormal",
+		"account_password": "YourPassword_123",
+		"account_type":     string("Normal"),
+	}
+	resourceId := "alicloud_polardb_account.default"
+	ra := resourceAttrInit(resourceId, basicMap)
+	serviceFunc := func() interface{} {
+		return &PolarDBService{testAccProvider.Meta().(*connectivity.AliyunClient)}
+	}
+	rc := resourceCheckInitWithDescribeMethod(resourceId, &v, serviceFunc, "DescribePolarDBAccount")
+	rac := resourceAttrCheckInit(rc, ra)
+	testAccCheck := rac.resourceAttrMapUpdateSet()
+	testAccConfig := resourceTestAccConfigFunc(resourceId, name, resourcePolarDBAccountConfigDependence)
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+
+		// module name
+		IDRefreshName: resourceId,
+
+		Providers:    testAccProviders,
+		CheckDestroy: rac.checkResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccConfig(map[string]interface{}{
+					"db_cluster_id":    "${alicloud_polardb_cluster.cluster.id}",
+					"account_name":     "tftestnormal",
+					"account_password": "YourPassword_123",
+					"account_type":     string("Normal"),
+				}),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck(nil),
+				),
+			},
+			{
+				ResourceName:            resourceId,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"account_password"},
+			},
+			{
+				Config: testAccConfig(map[string]interface{}{
+					"account_description": "from terraform",
+				}),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck(map[string]string{
+						"account_description": "from terraform",
+					}),
+				),
+			},
+			{
+				Config: testAccConfig(map[string]interface{}{
+					"account_password": "YourPassword_1234",
+				}),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck(map[string]string{
+						"account_password": "YourPassword_1234",
+					}),
+				),
+			},
+			{
+				Config: testAccConfig(map[string]interface{}{
+					"account_description": "tf test",
+					"account_password":    "YourPassword_123",
+				}),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck(map[string]string{
+						"account_description": "tf test",
+						"account_password":    "YourPassword_123",
+					}),
+				),
+			},
+		},
+	})
+
+}
+
+func resourcePolarDBAccountConfigDependence(name string) string {
+	return fmt.Sprintf(`
+	%s
+	variable "creation" {
+		default = "PolarDB"
+	}
+
+	variable "name" {
+		default = "%s"
+	}
+
+	variable "instancechargetype" {
+		default = "PostPaid"
+	}
+
+	variable "engine" {
+		default = "MySQL"
+	}
+
+	variable "engineversion" {
+		default = "8.0"
+	}
+
+	variable "instanceclass" {
+		default = "polar.mysql.x4.large"
+	}
+
+	resource "alicloud_polardb_cluster" "cluster" {
+		db_type = "${var.engine}"
+		db_version = "${var.engineversion}"
```

Comment:
Please remove the variables and write them directly in the resource

Reply:
> Please remove the variables and write them directly in the resource

done